packages/styles/package.json

```diff
@@ -18,30 +18,30 @@
     "linkDirectory": true
   },
   "scripts": {
-    "start": "npm-run-all --parallel styles:start schematics:start",
-    "styles:start": "gulp watch",
+    "start": "gulp watch",
     "prebuild": "pnpm clean",
-    "build": "npm-run-all --parallel styles:build schematics:build",
-    "styles:build": "gulp build",
+    "build": "gulp build",
     "unit": "gulp sass:tests",
     "clean": "rimraf out-tsc dist",
     "format": "prettier src/**/*.scss --write",
     "lint": "stylelint src/**/*.scss",
-    "lint:fix": "stylelint src/**/*.scss --fix",
-    "schematics:start": "copyfiles schematics/migrations.json dist && tsc -p tsconfig.schematics.json --watch",
-    "schematics:build": "copyfiles schematics/migrations.json dist && tsc -p tsconfig.schematics.json"
+    "lint:fix": "stylelint src/**/*.scss --fix"
+  },
+  "peerDependencies": {
+    "@ng-bootstrap/ng-bootstrap": "^15.0.0"
+  },
+  "peerDependenciesMeta": {
+    "@ng-bootstrap/ng-bootstrap": {
+      "optional": true
+    }
   },
   "dependencies": {
-    "@angular-devkit/core": "=15.0.4",
-    "@angular-devkit/schematics": "=15.0.4",
-    "@angular/core": "=15.0.4",
     "@popperjs/core": "2.11.8",
     "bootstrap": "5.3.2",
     "cheerio": "1.0.0-rc.12",
     "prettier": "3.1.0"
```

Comment:
These dependencies are also connected to the migrations, prettier for formatting output and cheerio for finding elements. I'm not sure if they are needed in the styles anymore, especially cheerio. Prettier could become a dev dependency so it's not delivered to projects.